misty_py/api.py

```diff
@@ -576,7 +577,13 @@ class MistyAPI(RestAPI):
 
     _pool = ThreadPoolExecutor(16)
 
-    def __init__(self, ip):
+    def __init__(self, ip=None):
+        if ip is None:
+            ip = os.environ.get("MISTY_IP")
+            if not ip:
+                raise Exception("You must provide an ip argument, or set $MISTY_IP")
+        if not ip.startswith("http://") or ip.startswith("https://"):
+            ip = "http://" + ip
```

Comment:
i'm going to make a small change here after merging. i prefer to keep my `__init__`s super clean, so i'm going to move this logic to a separate `staticmethod` but everything else looks good.